google_bigquery_writer/app.py

```diff
@@ -123,34 +126,31 @@ def action_run(self):
                 raise UserException(message)
         print('BigQuery Writer finished')
 
-    def action_list_projects(self):
```

Comment:
Tato metoda se stale pouziva
https://github.com/keboola/google-bigquery-writer/blob/8557106a0d7e3eaa2d7fe7242f44aec8bbdd26b6/google_bigquery_writer/app.py#L65-L70

Jestli chceme zachovat zpetnou kompatibilitu, tak to tu musi zustat vcetne testu.

Reply:
nikdo ty sync akce zatím nepoužívá, zpětná kompatibilita není třeba, vyrubu to odsud